operator/src/main/java/oracle/kubernetes/operator/steps/ManagedServersUpStep.java

```diff
@@ -294,13 +292,8 @@ private void logIfReplicasLessThanClusterServersMin(WlsClusterConfig clusterConf
       }
     }
 
-    private void addValidationErrorEventAndWarning(String msgId, Object... messageParams) {
-      LOGGER.warning(msgId, messageParams);
-      String message = LOGGER.formatMessage(msgId, messageParams);
-      if (!skipEventCreation) {
-        eventStep = createEventStep(new EventData(EventItem.DOMAIN_VALIDATION_ERROR, message));
-      }
-      info.addValidationWarning(message);
+    private void addReplicasTooHighValidationErrorEventAndWarning(Object... messageParams) {
+      LOGGER.warning(MessageKeys.REPLICAS_EXCEEDS_TOTAL_CLUSTER_SERVER_COUNT, messageParams);
```

Comment:
This method no longer emits an event. Rename the method?

Reply:
renamed.